yun/libraries/Parse/examples/test/Test.ino

```diff
@@ -0,0 +1,111 @@
+#include <Parse.h>
+#include <Bridge.h>
+#include <assert.h>
+
+/***** Integration Tests of Parse Arduino Yún SDK *****/
+ParseClient client;
+int i = 0; // testId for a specific test
+
+void basicObjectTest() {
+  Serial.println("basic object operation test");
+
+  Serial.println("create...");
+  ParseObjectCreate create;
+  create.setClassName("Temperature");
+  create.add("temperature", 175.0);
+  create.add("leverDown", true);
+  ParseResponse createResponse = create.send();
+  char* objectId = new char[10];
+  strcpy(objectId, createResponse.getString("objectId"));
+  assert(createResponse.getErrorCode() == 0);
+  createResponse.close();
+
+  Serial.println("update...");
+  ParseObjectUpdate update;
+  update.setClassName("Temperature");
+  update.setObjectId(objectId);
+  update.add("temperature", 100);
+  ParseResponse updateResponse = update.send();
+  assert(updateResponse.getErrorCode() == 0);
+  updateResponse.close();
+
+  Serial.println("get...");
+  ParseObjectGet get;
+  get.setClassName("Temperature");
+  get.setObjectId(objectId);
+  ParseResponse getResponse = get.send();
+  double temp = getResponse.getDouble("temperature");
+  assert(temp == 100);
+  getResponse.close();
+
+  Serial.println("delete...");
+  ParseObjectDelete del;
+  del.setClassName("Temperature");
+  del.setObjectId(objectId);
+  ParseResponse delResponse = del.send();
+  delResponse.close();
+
+  Serial.println("test passed\n");
+}
+
+void objectDataTypesTest() {
+  Serial.println("data types test");
+
+  ParseObjectCreate create;
+  create.setClassName("TestObject");
+  create.addGeoPoint("location", 40.0, -30.0);
+  create.addJSONValue("dateField", "{ \"__type\": \"Date\", \"iso\": \"2011-08-21T18:02:52.249Z\" }");
+  create.addJSONValue("arrayField", "[ 30, \"s\" ]");
+  create.addJSONValue("emptyField", "null");
+  ParseResponse createResponse = create.send();
+  assert(createResponse.getErrorCode() == 0);
+  createResponse.close();
+
+  Serial.println("test passed\n");
+}
+
+void queryTest() {
+  Serial.println("query test");
+
+  ParseQuery query;
```

Comment:
Does this assume the server already has this object?  If so, could you also check in a CSV file that people can import into their app via Parse.com Data Browser before running tests?